app/workers/package_manager_download_worker.rb

```diff
@@ -61,9 +67,13 @@ def perform(platform_name, name, version = nil, source = "unknown")
     project = platform.update(name, sync_version: sync_version)
 
     # Raise/log if version was requested but not found
-    if version.present? && !Version.exists?(project: project, number: version)
+    if version.present? && true # !Version.exists?(project: project, number: version)
```

Comment:
Do we need to add this conditional back in? With it commented out it will re queue every time right?